src/api/routes/summaries.ts

```diff
@@ -77,6 +78,42 @@ const list = procedure
 
   checkPermissionForGroup(ctx.user, t.group);
 
+  // find all handlebar from summaries under one transcript
+  let handlebars: string[] = [];
+  for (let s of t.summaries) {
+    const matches = s.summary.match(/{{(.*?)}}/g);
+    // parse and find handlebar values but get rid of the repeating values
+    if (matches) {
+       handlebars = [...handlebars, ...matches.map(match => match.slice(2, -2).trim())]
+        .reduce((acc, cur) => acc.includes(cur) ? acc : [...acc, cur], handlebars);
```

Comment:
this line is too cryptic. break it down into more readable smaller lines.

also, use this instead for simplicity:

````
const handlebar = ...

handlebars.push(...matches)
```